infra/charts/feast-feature-server/values.yaml

```diff
@@ -33,7 +36,7 @@ securityContext: {}
 
 service:
   type: ClusterIP
-  port: 80
+  port: 6566 # Port needs to be changed according to feast mode. For example, for offline mode, the default port is [8815].
```

Comment:
this should not be changed as this is not the intention of this helm value. it would also likely break existing chart users